config/locales/en.yml

```diff
@@ -868,36 +1595,108 @@ en:
       subscribe: Subscribe
       subscribed_on_another_device: Subscribe on this Device
       unsubscribe: Unsubscribe
+  controllers:
+    ApplicantsController:
+      link_expired: That one-time link has expired, or is invalid. If you have already completed enrollment, please sign in.
+      welcome: Welcome to %{school_name}!
+    CoursesController:
+      errors: "There were errors with your submission: %{form_errors}"
+      sent_mail: We've sent you a verification mail. It should reach you in less than a minute. Click the link in the email to sign up, and get started.
+    Schools__CoursesController:
+      teams_active: Teams marked active successfully!
+    UsersController:
+      link_expired: That link has expired or is invalid. Please try again.
+    Users__OmniauthCallbacksController:
+      add_facebook: Please <a href="https://www.facebook.com/settings?tab=applications" target="_blank" rel="noopener">remove \'Pupilfirst\' from your authorized apps list</a> and try signing in again.
+      add_github: Please <a href="https://github.com/settings/profile" target="_blank" rel="noopener">add a public email address to your Github profile</a> and try again.
+      add_other: Please sign in using another method.
+      denied: Authentication was denied. Please try again.
+      denied_by: Authentication was denied by %{provider}. Please try again.
+      email_unregistered: "Your email address: %{email} is unregistered."
+      not_receive_email: "We're sorry, but we did not receive your email address from %{provider_name}. "
+    Users__SessionsController:
+      already_signed: You are already signed in.
+      link_expired: That one-time link has expired, or is invalid. Please try signing in again.
+      link_used: That one-time link has already been used, or is invalid. Please try resetting your password again.
   courses:
```

Comment:
@prashanth726 This being controller keys should go under the new `controllers` namespace, right?

Reply:
@harigopal - Yes, it is already in the controller namespace.  Did I miss something here?  🤔 
https://github.com/indicozy/op-onai-edu/blob/0922c691b3601d68b445ac3775ee30bb415a7d1f/config/locales/en.yml#L1627

Reply:
@prashanth726 L1634 has the key `courses:` at an indentation of 2 spaces. L1200 has the key `controllers:` at an indentation of 2 spaces.

They're keys at the same level.

Also, there's a different key `courses:` inside `controllers:` at L1606.